migration_original/ODS1Stage/tables/Base/Degree/spu_translated_Degree.sql

```diff
@@ -30,22 +30,22 @@ begin
 ---------------------------------------------------------     
 
 select_statement := $$
-                    with CTE_degrees as (
-                        select distinct json.degree_DegreeCode as DegreeCode
-                        from raw.vw_PROVIDER_PROFILE as JSON
-                        where not exists (
-                            select d.degreeabbreviation
-                            from base.degree as d
-                            where d.degreeabbreviation = json.degree_DegreeCode
-                        ) and DegreeCode is not null
+                    with Cte_degree as (
+                        select
+                            distinct
+                            to_varchar(json.value:DEGREE_CODE) as DegreeCode,
+                            MAX(to_varchar(json.value:UPDATED_DATETIME)) as LastUpdateDate
+                        from $$ || mdm_db || $$.mst.provider_profile_processing as p
+                        , lateral flatten(input => p.PROVIDER_PROFILE:DEGREE) as json
+                        group by 
```

Comment:
not a fan of a distinct with a group by, it's one or the other imo

Reply:
Changed